sid/index.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
```suggestion
```

IE browser has reached end of life as per https://docs.microsoft.com/en-us/lifecycle/products/internet-explorer-11. Unless you're working a web application that is required to work on IE, the UA-Compatible meta tag can be deleted.